ginga/rv/plugins/Cuts.py

```diff
@@ -173,7 +182,11 @@ def build_gui(self, container):
 
         self.slit_plot = plots.Plot(logger=self.logger,
                                     width=400, height=400)
-        self.slit_plot.add_axis(axisbg='black')
+        if plots.MPL_GE_2_0:
```

Comment:
@pllim, do we need to check for this identifier with a `hasattr()` as well?  I imagine it does not exist too far back into the 1.x series.